.github/workflows/verify.yml

```diff
@@ -302,6 +302,7 @@ jobs:
       uses: actions/download-artifact@v2
       with:
         name: archive-src
+
```

Comment:
Please don't make unrelated stylistic changes.

The reason why there is no empty line is that those two steps go "hand-in-hand" download + extract the source.
One can argue whether or not an empty line should be here, however, that would be a separate commit changing that.
If you feel strongly about the it, then the best way forward would be the addition of some tool forcing certain formatting for the ``.yaml`` or at least a separate commit that one can add to git-blame-ignore.